examples/grpc/capitalize_client.js

```diff
@@ -0,0 +1,95 @@
+/**
+ * Copyright 2019, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      gRPC://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const path = require('path');
+const grpc = require('grpc');
+const protoLoader = require('@grpc/proto-loader');
+const tracing = require('@opencensus/nodejs');
+const { plugin } = require('@opencensus/instrumentation-grpc');
+const { StackdriverTraceExporter } =
+    require('@opencensus/exporter-stackdriver');
+
+let tracer;
+
+setupOpencensusAndExporters();
+
+const PROTO_PATH = path.join(__dirname, 'protos/defs.proto');
+const PROTO_OPTIONS = { keepCase: true, enums: String, defaults: true, oneofs: true };
+const definition = protoLoader.loadSync(PROTO_PATH, PROTO_OPTIONS);
+const rpcProto = grpc.loadPackageDefinition(definition).rpc;
+
+function main () {
+  const client = new rpcProto.Fetch('localhost:50051',
+    grpc.credentials.createInsecure());
+  let data;
+  if (process.argv.length >= 3) {
+    data = process.argv[2];
+  } else {
+    data = 'opencensus';
+  }
+  console.log('> ', data);
+
+  tracer.startRootSpan({ name: 'octutorialsClient.capitalize' }, rootSpan => {
+    client.capitalize({ data: Buffer.from(data) }, function (err, response) {
+      if (err) {
+        console.log('could not get grpc response');
+        return;
+      }
+      console.log('< ', response.data.toString('utf8'));
+      rootSpan.end();
+    });
+  });
+
+  setTimeout(() => {
+    console.log('done.');
```

Comment:
What's the role of this message / is there a better way to check for doneness than waiting one minute?

Reply:
The minimum reporting period for Stackdriver is 1 minute. The thread with the ```StackdriverStatsExporter``` must live for at least the interval past any metrics that must be collected, or some risk being lost if they are recorded after the last export. We planned to add a ```flush``` method instead of doing this.

I will enable ```StackdriverStatsExporter``` here, once the gRPC stats are available #270

Reply:
OK, interesting. Can you add a comment to explain this?

Reply:
Done.